utilities/tools/helper/Get-PipelineStatusUrl.ps1

```diff
@@ -0,0 +1,114 @@
+﻿<#
+.SYNOPSIS
+Generate the status URL for GitHub module action workflows
```

Comment:
I understand this is now generalized for both ADO and GH and for both module and platform pipelines. If that is the case:
```suggestion
Generate the status URL for the given pipeline/workflow file
```